modules/ROOT/pages/browser-support.adoc

```diff
@@ -12,11 +12,18 @@ Use Anypoint Platform with the latest version of any supported browser:
 * Mozilla Firefox
 * Safari
 
-The Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
+Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
```

Comment:
Browser support for Anypoint Platform follows the Salesforce Lightning Experience policy.